### PR TITLE
Added lambda for approving the ticket

### PR DESCRIPTION
**Pull request by athiruma**
1. Added the lambda function for manager auto-approve requests.
2. Not uploading to the PyPI. Wrote for special purpose

Note: Not sending the approved mail to the user, highly recommend using the site to approve this request.

### CODE REVIEW

File: aws_lambda_functions/CloudResourceOrchestration/lambda_function.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                        'from': ses_data.get('mail', {}).get('source'),
+                        'to': common_headers.get('to'),
+                        'subject': common_headers.get('subject')
+                    })
+    return mail_details
+
+
+def lambda_handler(event, context):
+    """
+    This lambda function is to approve the user budget by Email Receiving
```
Comment (ebattat) [inline aws_lambda_functions/CloudResourceOrchestration/lambda_function.py:40]: This lambda method approves

File: aws_lambda_functions/CloudResourceOrchestration/lambda_function.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
+import json
```
Comment (ebattat) [inline aws_lambda_functions/CloudResourceOrchestration/lambda_function.py:1]: Why did you call it lambda function ?

File: aws_lambda_functions/CloudResourceOrchestration/lambda_function.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+import boto3
+import jira
+
+ssm_client = boto3.client('ssm', region_name='us-east-1')
```
Comment (ebattat) [inline aws_lambda_functions/CloudResourceOrchestration/lambda_function.py:5]: I think that it should be under class, put all the parameters under class

**Review by ebattat — APPROVED** [review]

File: aws_lambda_functions/CloudResourceOrchestration/lambda_function.py (hunk 2 above)
Comment (athiruma) [reply aws_lambda_functions/CloudResourceOrchestration/lambda_function.py:1]: It is written in the lambda function document. I followed it.

File: aws_lambda_functions/CloudResourceOrchestration/lambda_function.py (hunk 3 above)
Comment (athiruma) [reply aws_lambda_functions/CloudResourceOrchestration/lambda_function.py:5]: yes, right.
Here in the lambda case, we always need to reduce the execution time of lambda. So I thought to write it on top, to reduce the calls.

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**